src/app/gear-power/GearPower.scss

```diff
@@ -0,0 +1,137 @@
+.gearPowerSheet {
```

Comment:
In general I prefer CSS modules for new stuff but it isn't a big deal.

Reply:
artifact of stealing code from infusion finder. fixed now.